comment_scan/helpers.rb

```diff
@@ -121,9 +121,9 @@ def perspective_scan(text, perspective_key: '', perspective_log: Logger.new('/de
   end
 end
 
-def percent_str(numerator, denominator)
-  return '-' if denominator.zero?
-  "#{(numerator*100.0/denominator).round(8)}%"
+def percent_str(numerator, denominator, precision=8, blank_str='-')
```

Comment:
I'd recommend kwargs instead of default values for these arguments, because the order is unimportant.